akka-remote/src/main/java/akka/remote/artery/fastutil/README-fastutil.md

```diff
@@ -0,0 +1,14 @@
+Instructions to re-create fast util classes
+-------------------------------------------
+
+Fast Util provides type specific optimised collections
+License: Apache v2
+Website: http://fastutil.di.unimi.it/
+
+Process to obtain sources:
+
+```
+git clone https://github.com/vigna/fastutil
+cd fastutil
+make sources # takes around a minute
+```
```

Comment:
so the FastUtil we have here is basically a by-hand ripped out minimal subset of those 3000 classes implementing maps, that allows to use Object2IntMap